src/Pokemon.java

```diff
@@ -0,0 +1,49 @@
+public abstract class Pokemon {
+    private final String name;
+    private int hp;
+    private final int level;
+    private final String food;
+    private final String sound;
+    private final String type;
+
+    public Pokemon(String name, int level, int healthPoints, String type, String speech, String food) {
+
+        this.name = name;
+        this.level = level;
+        this.hp = healthPoints;
```

Comment:
Je hoeft hier geen this. te gebruiken (maar het mag wel), omdat er geen naamsverwarring is.